packages/config/config/devices/0x0090/893.json

```diff
@@ -0,0 +1,175 @@
+{
+	"manufacturer": "Spectrum Brands",
+	"manufacturerId": "0x0090",
+	"label": "893",
+	"description": "10 Button Z-Wave Plus v2 Deadbolt",
+	"devices": [
+		{
+			"productType": "0x0811",
+			"productId": "0x13a8",
+			"zwaveAllianceId": 4184
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Status LED Control",
+			"description": "The configuration parameter 1 is a read/write parameter with a one-byte field, used to control the status led. if set to on, the status LED will show the status of the latch every 6 seconds. if set to off, the status LED will not be shown",
```

Comment:
This description seems unnecessary. If anything you might mention the 6 second interval.